test/evergreen.yml

```diff
@@ -3534,32 +3545,24 @@ tasks:
             done
 
   - name: split-stress-test
-    tags: ["stress-test-1", "stress-test-zseries-1"]
+    tags: ["stress-test-1", "stress-test-ppc-1", "stress-test-zseries-1"]
+    # Set 2.5 hours timeout (60 * 60 * 2.5)
+    exec_timeout_secs: 9000
```

Comment:
Is the setting here to make the task fail (timeout) fast? 
The [project-level setting for this variable](https://github.com/wiredtiger/wiredtiger/blob/cb650185eb72177cc0b1cae7c83b598bdb08e020/test/evergreen.yml#L24) is 6 hrs.

Reply:
Yes to fail fast as we have zSeries and PPC running every 2 hours and we don't want to hold that machines.

Reply:
Got it.